doc/source/categorical.rst

```diff
@@ -182,6 +182,19 @@ It's also possible to pass in the categories in a specific order:
     indicate an ordered ``Categorical``.
 
 
+.. note::
+
+    Pay attention! Result of ``.unique()`` not always the same as ``Series.cat.categories``,
```

Comment:
Don't need the "Pay attention". Add a "The" before "result". Let's change ``.unique()`` to ``Series.unique()`` to make that a bit clearer.